src/editor/contentState/selectionCtrl.js

```diff
@@ -0,0 +1,43 @@
+const selectionCtrl = ContentState => {
+  ContentState.prototype.tableCellHandler = function (event) {
+    const { start, end } = this.cursor
+    const startBlock = this.getBlock(start.key)
+    const endBlock = this.getBlock(end.key)
+    const { type: startType } = startBlock
+    const { type: endType } = endBlock
+    if (/th|td/.test(startType) && /th|td/.test(endType)) {
+      event.preventDefault()
+      event.stopPropagation()
+      if (start.key === end.key) {
+        const { text, key } = startBlock
+        this.cursor = {
+          start: { key, offset: 0 },
+          end: { key, offset: text.length }
+        }
+      } else {
```

Comment:
maybe start block and and block are in two tables? if they are in two tables, just select all the document?